README.md

```diff
@@ -1,68 +1,109 @@
 # drone-pypi
 
 [![Build Status](http://beta.drone.io/api/badges/drone-plugins/drone-pypi/status.svg)](http://beta.drone.io/drone-plugins/drone-pypi)
+[![Coverage Status](https://aircover.co/badges/drone-plugins/drone-pypi/coverage.svg)](https://aircover.co/drone-plugins/drone-pypi)
 [![](https://badge.imagelayers.io/plugins/drone-pypi:latest.svg)](https://imagelayers.io/?images=plugins/drone-pypi:latest 'Get your own badge on imagelayers.io')
 
-Drone plugin for publishing to the Python package index
+Drone plugin to publish files and artifacts to Python package index
 
-## Usage
+## Binary
 
-Upload a source distribution to PyPI
+Build the binary using `make`:
 
-```sh
-./drone-pypi <<EOF
-{
-	"workspace": {
-		"path": "/drone/my-module-py"
-	}
-	"vargs": {
-		"username": "guido",
-		"password": "secret"
-	}
-}
-EOF
 ```
-
-Upload a source distribution and a wheel to PyPI
-
-```sh
-./drone-pypi <<EOF
-{
-	"workspace": {
-		"path": "/drone/my-module-py"
-	}
-	"vargs": {
-		"distributions": ["sdist", "bdist_wheel"],
-		"username": "guido",
-		"password": "secret"
-	}
-}
-EOF
+make deps build
 ```
 
-Upload a source distribution to a private PyPI server, e.g. [simplepypi][]
+### Example
 
 ```sh
 ./drone-pypi <<EOF
 {
-	"workspace": {
-		"path": "/drone/my-module-py"
-	}
-	"vargs": {
-		"repository": "https://pypi.example.com"
-	}
+    "repo": {
```

Comment:
It is, but a huge chunk (if not the majority) of this plugin's users will be pushing to their own PyPi-compatible repositories. It's an important argument to show because of this.